Core/Config.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2012- PPSSPP Project.
+﻿// Copyright (c) 2012- PPSSPP Project.
```

Comment:
Tried [EditorConfig Language Service](http://vsixgallery.com/extension/1209461d-57f8-46a4-814a-dbe5fecef941/) as well as [Fix File Encoding](https://vlasovstudio.com/fix-file-encoding/) - editing it's list of protected files - neither helped:c. The description of that latter extension would suggests that it was doing the same thing in older VS, however I don't remember ever having that problem:|.
 At least manual solution described there seems to work ~ that is "saving as" with "UTF-8 without signature" encoding ~ a bit annoying through.